### PR TITLE
Update nuget packages

### PR DESCRIPTION
**Pull request by Yury-Fridlyand**
Signed-off-by: Yury-Fridlyand <yuryf@bitquilltech.com>

### Description
Update some Nuget packages which had dependencies with critical CVEs reported.
See https://github.com/opensearch-project/opensearch-net/pull/88 for more info about security update.

### Issues Resolved
`CVE-2021-26701` in `System.Text.Encodings.Web`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by MaxKsyunz — APPROVED** [review]

**Review by joshuali925 — APPROVED** [review]

File: src/OpenSearch.OpenSearch.Xunit/XunitPlumbing/SkippingTestCase.cs [hunk 1]
```diff
@@ -45,9 +45,11 @@ public SkippingTestCase()
 		/// <param name="testMethod">The test method this test case belongs to.</param>
 		/// <param name="testMethodArguments">The arguments for the test method.</param>
 		public SkippingTestCase(string skipReason, ITestMethod testMethod, object[] testMethodArguments = null)
-			: base(TestMethodDisplay.ClassAndMethod, testMethod, testMethodArguments) =>
+			: base(TestMethodDisplay.ClassAndMethod, TestMethodDisplayOptions.None, testMethod, testMethodArguments) =>
 			SkipReason = skipReason ?? "skipped";
 
+		public int Timeout => 0;
```
Comment (dblock) [inline src/OpenSearch.OpenSearch.Xunit/XunitPlumbing/SkippingTestCase.cs:51]: This looks new, why was it added?

File: src/OpenSearch.Stack.ArtifactsApi/OpenSearch.Stack.ArtifactsApi.csproj [hunk 2]
```diff
@@ -7,7 +7,9 @@
     
     <ItemGroup>
         <PackageReference Include="SemanticVersioning" Version="0.8.0" />
-        <PackageReference Include="System.Text.Json" Version="4.6.0" />
+        <PackageReference Include="System.Net.Http" Version="4.3.4" />
+        <PackageReference Include="System.Runtime.InteropServices.RuntimeInformation" Version="4.3.0" />
```
Comment (dblock) [inline src/OpenSearch.Stack.ArtifactsApi/OpenSearch.Stack.ArtifactsApi.csproj:11]: This looks added, why?

**Review by dblock — CHANGES_REQUESTED** [review]
I see some unexpected things looking carefully at https://github.com/opensearch-project/opensearch-net-abstractions/pull/19/files?w=1

File: src/OpenSearch.OpenSearch.Xunit/XunitPlumbing/SkippingTestCase.cs (hunk 1 above)
Comment (Yury-Fridlyand) [reply src/OpenSearch.OpenSearch.Xunit/XunitPlumbing/SkippingTestCase.cs:51]: `xunit` was updated and interface `TestMethodTestCase` got a new method `Timeout` which should be overloaded.

File: src/OpenSearch.Stack.ArtifactsApi/OpenSearch.Stack.ArtifactsApi.csproj (hunk 2 above)
Comment (Yury-Fridlyand) [reply src/OpenSearch.Stack.ArtifactsApi/OpenSearch.Stack.ArtifactsApi.csproj:11]: These are essential packages, compilation fails without them.
I'm also surprised how it was working before.

Comment (dblock) [reply src/OpenSearch.Stack.ArtifactsApi/OpenSearch.Stack.ArtifactsApi.csproj:11]: Eh 🤔

**Review by dblock — APPROVED** [review]

**Pull request merged**